ParkingZoneApp.Tests/ModelValidationTests/DetailVMTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿using ParkingZoneApp.Models;
+using ParkingZoneApp.ViewModels.ParkingZones;
+using System.ComponentModel.DataAnnotations;
+
+namespace ParkingZoneApp.Tests.ModelValidationTests
+{
+    public class DetailVMTests
+    {
+        public static IEnumerable<object[]> testDetailsVMData =>
+           new List<object[]>
+           {
+                new object[] { Guid.NewGuid(), null, "Chilonzor", new DateOnly(2024, 4, 12) },
+                new object[] { null, "7Parking", "Qoraqamish", new DateOnly(2024, 4, 12) },
+                new object[] { Guid.NewGuid(), "Sharafshon", null, new DateOnly(2024, 4, 12) },
+                new object[] { Guid.NewGuid(), "Sharafshon", "Andijon", null }
+           };
+
+        [Theory]
+        [MemberData(nameof(testDetailsVMData))]
+        public void CreateVM_WithValidData_ShouldPassValidation(Guid? id, string name, string address, DateOnly? createdDate)
+        {
+            //Arrange
+            ParkingZone parkingZone = new ParkingZone()
```

Comment:
unused variable, where are u using this variable, just create detailsVM and assign to it

Reply:
I have to pass that object to detailsVM as a parameter
![image](https://github.com/dilmurod-rozikov/ParkingZone/assets/141717735/80510e3a-71c2-40cb-88e7-0b13588eb034)

Reply:
done